recipes/libjxl/all/conanfile.py

```diff
@@ -0,0 +1,101 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class JpegxlConan(ConanFile):
+    name = "libjxl"
```

Comment:
```suggestion
    name = "jpeg-xl"
```

Follow the original name

Reply:
This is a non-obvious part. In the repository it is said that the name of the library is `libjxl`. I thought it would be logical to call the recipe `libjxl` as well. Also, almost all image libraries have similar names: `libjpeg`, `libwebp` etc. If it is best to change the name to `jpeg-xl`, then I won't mind. I just think that `libjxl` is a bit more accurate.

Reply:
This is not first project which uses a different name from library name here in CCI. I can create a project which provides a library named libjxl too, but jpeg-xl is unique.